src/helpers/determineLanguage.ts

```diff
@@ -0,0 +1,22 @@
+import { catalogs } from "config/i18n";
+
+// input could be in both "en" and "en-US" formats
+export function determineLanguage(input: string): string {
+  let language;
+
+  const [languageFromInput, countryFromInput] = input.split("-");
+
+  if (input in catalogs) {
+    language = input;
+  } else if (languageFromInput in catalogs) {
+    language = languageFromInput;
+  } else if (countryFromInput && countryFromInput === "AR") {
```

Comment:
```suggestion
  } else if (countryFromInput === "AR") {
```

Reply:
I like your suggestion, but I actually thought about leaving the check like this to make it explicit for other devs in any future change that `countryFromInput` can be `undefined` depending on how the wallet is serving it